ADA/databricks_rstudio_sql_warehouse.qmd

```diff
@@ -1,105 +1,210 @@
 ---
-  title: "Setup Databricks SQL Warehouse with RStudio"
+  title: "Connecting to Databricks from RStudio"
 ---
 
-<p class="text-muted">The following instructions set up an ODBC connection between your laptop and your DataBricks SQL warehouse, which can then be used in R/RStudio to query data using an ODBC based package.
-SQL Warehouses are able to run SQL and can be used within the DataBricks environment, or through RStudio to run SQL code. The `sparklyr` package can also be used from within RStudio (only) as it converts the request for data to Spark SQL behind the scenes.</p>
+The following instructions will help you to set up a connection between your laptop and your Databricks SQL warehouse or personal cluster, which can then be used in RStudio to query data.
+
+You can use data from Databricks in two different ways:
+
+-   In the SQL editor or in notebooks via the Databricks environment
+-   In RStudio via an ODBC connection, similarly to the way that you might currently use SQL Server
 
 ------------------------------------------------------------------------
 
-# Pre-requisites
+## Compute resources
+
+When your data is moved to Databricks, it will be stored in the Unity Catalog and you will need to use a compute resource to access it from other software such as RStudio.
+
+A compute resource allows you to run your code using cloud computing power instead of using your laptop's processing power. This means that using compute resources can allow your code to run faster than it would if you ran it locally, as it is like using the processing resources of multiple computers at once. On this page, we will be referring to the use of SQL Warehouses as the compute resource to run your code. 
```

Comment:
Are we only referring to SQL warehouses as the compute resource in this page? It covers a bit of both and it's equally applicable to both.

Reply:
Why is that given the process is identical? It seems like a lot of needless repetition.

Reply:
The process is, but the background info is not